packages/frontend-near-wallet/src/modules/wallet-migration/components/MigrationSecret.tsx

```diff
@@ -0,0 +1,104 @@
+
+import React, { useCallback, useState } from 'react';
+import Button from '../../../components/Button';
+import { styled } from '../../../styles';
+import { getMyNearWalletUrl } from '../../../utils/constants';
+
+import MyNearWalletLogo from '../../../assets/images/ImgMyNearWalletLogo';
+import ClickToCopy from '../../../components/ClickToCopy';
+
+const MigrationSecret = ({ secretKey, handleCancel, showMigrateAccount }) => {
+    const [shouldContinueDisabled, setContinueDisabled] = useState(true);
+
+    const setContinueEnable = useCallback(() => {
+        setContinueDisabled(false);
+    }, []);
+
+    return (
+        <StyledContainer>
+            <MyNearWalletLogo className="logo" />
+            <h3 className='ttl'>Password required to import accounts</h3>
+            <p className='desc'>
+                You'll need this <strong>password</strong> to securely import your accounts to MyNearWallet. Copy or write it down until you've completed transferring your accounts.
+            </p>
+            <ClickToCopy
```

Comment:
I think there should be a clear call to action button to copy the the password. If the user is on a touch screen device, they won't have any context as to why they can not hit the `continue` button.

Either, hitting `continue` button should prompt them with a popup that says something like 'I have copied down the password.' and have a button for `cancel` and `continue`

or the continue button should be in a disabled state, until the user copies the password and there should definitely be a copyToClipboard icon on the bottom right corner so the user does not have to guess that clicking the field copies to clipboard. 

![wallet-test](https://user-images.githubusercontent.com/25015977/184192768-10d8ca93-195c-4416-9fa6-67263a218f53.gif)